config/Dockerfiles/linchpin-install.sh

```diff
@@ -7,7 +7,7 @@ make pip_install.sh
 ./pip_install.sh
 popd
 
-pip install .
+pip install . --ignore-installed
```

Comment:
Should you add `--ignore-installed` to the other `pip install` commands as well?

Reply:
I would like to just overwrite the packages for linchpin.
Though it is safe in the container I would not recommend it since there can be accidental overwrites.